inc/db.php

```diff
@@ -155,7 +155,7 @@ public static function findByID($id, $version = null) {
 	private static function parseOrderBy($order_by) {
 		$valid_sort_columns = [
 			'downloadcount' => 'packages.DownloadCount',
-			'id' => 'packages.PackageId',
+			'id' => 'LOWER(packages.PackageId)',
```

Comment:
I wonder if we should just store this in lowercase. This is fine for now though. Thanks!

Reply:
I did wonder about this, I struggled as to either have the column in the table as case-insensitive or make all references to PackageId lower case, this would have to include the file system folder names.  This fix had the least impact on existing systems, users would probably have to push all their packages to the server again if stored as lowercase.

Looking at the %userprofile%\.nuget\packages folder on a Windows machine I can see that the file system is in lowercase.

When I was searching for documentation for NuGet I did come across something that said that NuGet is case-insensitive, to be found again.

I might take another look at this in the future and fix for lowercase storage.

Thanks (and thanks for the project, it's very helpful for me with my internal build server)